### PR TITLE
OSDOCS-1509 change pruner loglevel

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1509

### CODE REVIEW

File: modules/pruning-images.adoc [hunk 1]
```diff
@@ -66,8 +67,13 @@ status:
...
 <10> `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
 <11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
+<12> `loglevel`: Sets the level of log output for the pruner job. Defaults to  `Normal`.
+* `Normal`
+* `Debug`
+* `Trace`
+* `TraceAll`
+<13>`observedGeneration`: The generation observed by the Operator.
```
Comment (dmage) [inline modules/pruning-images.adoc:75]:
```suggestion
<13> `observedGeneration`: The generation observed by the Operator.
```

File: modules/pruning-images.adoc [hunk 2]
```diff
@@ -36,9 +36,10 @@ spec:
   startingDeadlineSeconds: 60 <9>
   successfulJobsHistoryLimit: 3 <10>
   failedJobsHistoryLimit: 3 <11>
+  loglevel: TraceAll <12>
```
Comment (dmage) [inline modules/pruning-images.adoc:39]:
```suggestion
  logLevel: TraceAll <12>
```

File: modules/pruning-images.adoc [hunk 3]
```diff
@@ -66,8 +67,13 @@ status:
 <9> `startingDeadlineSeconds`: Start deadline for `CronJob`. This is an optional field.
 <10> `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
 <11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
+<12> `loglevel`: Sets the level of log output for the pruner job. Defaults to  `Normal`.
```
Comment (dmage) [inline modules/pruning-images.adoc:70]:
```suggestion
<12> `logLevel`: Sets the level of log output for the pruner job. Defaults to  `Normal`.
```

File: modules/pruning-images.adoc (hunk 2 above)
Comment (dmage) [reply modules/pruning-images.adoc:39]: afaik it's case sensitive

File: modules/pruning-images.adoc [hunk 4]
```diff
@@ -36,9 +36,10 @@ spec:
   startingDeadlineSeconds: 60 <9>
   successfulJobsHistoryLimit: 3 <10>
   failedJobsHistoryLimit: 3 <11>
+  logLevel: TraceAll <12>
```
Comment (wzheng1) [inline modules/pruning-images.adoc:39]: spec:
  logLevel: Normal

The default value is Normal, could we change to Normal?

**Comment by wzheng1** [comment]
Comments added.

File: modules/pruning-images.adoc (hunk 4 above)
Comment (bmcelvee) [reply modules/pruning-images.adoc:39]: Certainly! Updating now. Thanks!

**Comment by wzheng1** [comment]
Acked by QE.

File: modules/pruning-images.adoc [hunk 5]
```diff
@@ -66,8 +67,13 @@ status:
 <9> `startingDeadlineSeconds`: Start deadline for `CronJob`. This is an optional field.
 <10> `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
 <11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
+<12> `logLevel`: Sets the level of log output for the pruner job. Defaults to  `Normal`.
```
Comment (pneedle-rh) [inline modules/pruning-images.adoc:70]: You could shorten this to the following:

 <12> `logLevel`: Sets the log level for the pruner job. Defaults to  `Normal`.

File: modules/pruning-images.adoc [hunk 6]
```diff
@@ -66,8 +67,13 @@ status:
...
+* `Normal`
+* `Debug`
+* `Trace`
```
Comment (pneedle-rh) [inline modules/pruning-images.adoc:73]: I think that a small description about each log level would be useful for the reader. If I were setting `Trace` or `TraceAll` log levels in my cluster I would like to know what the difference is between the two. In a critical situation this could help prevent multiple reproducers from being required in order to debug something.

**Review by pneedle-rh — COMMENTED** [review]
@bmcelvee I have now completed my review. I have added a couple of minor comments. Otherwise, this looks good to me!

File: modules/pruning-images.adoc (hunk 6 above)
Comment (bmcelvee) [reply modules/pruning-images.adoc:73]: Agreed, and thank you. I checked the API and they aren't defined, so I'm not sure how to describe them at this point. I have a related issue/PR, so I'll merge this one as is an update the descriptions as soon as I get the information.
